cli/genpolicy/genpolicy.go

```diff
@@ -74,6 +74,20 @@ func (r *Runner) Run(ctx context.Context, yamlPath string, logger *slog.Logger)
 	return nil
 }
 
+func env() []string {
+	env := []string{"RUST_LOG=info", "RUST_BACKTRACE=1"}
```

Comment:
I'm wondering if we should just pass `os.Environ()? This way we would also support other config Envs like `HTTP_PROXY` should genpolicy use it at some point.
We can still have default values for `RUST_LOG` and `RUST_BACKTRACE` though.

Reply:
Done